presto-main/src/main/java/com/facebook/presto/cost/ScalarStatsCalculator.java

```diff
@@ -96,5 +105,37 @@ protected ColumnStatistics visitLiteral(Literal node, Void context)
             builder.setNullsFraction(Estimate.zeroValue());
             return builder.build();
         }
+
+        protected ColumnStatistics visitCast(Cast node, Void context)
+        {
+            // todo - are there cases that lowValue/highValue does not map to lowValue/highValue after cast?
```

Comment:
I think cast from `varchar` to `cast` is like that, e.g:
`cast('a' as char(2))`,
casted `a` is not a min anymore
Previously `a` < `a\0`, but
after cast it is no longer true, e.g:
`a_` > `a\0`